database/rrdhost.c

```diff
@@ -838,7 +838,7 @@ struct label *load_auto_labels()
             add_label_to_list(label_list, "_virt_detection", localhost->system_info->virt_detection, LABEL_SOURCE_AUTO);
 
     label_list = add_label_to_list(
-        label_list, "_is_master", (localhost->next || configured_as_master()) ? "true" : "false", LABEL_SOURCE_AUTO);
+        label_list, "_is_parent", (localhost->next || configured_as_parent()) ? "true" : "false", LABEL_SOURCE_AUTO);
```

Comment:
What's the impact of this to compatibility?

Reply:
It affects google-analytics but they are converted to indices before hitting the endpoint and nobody else uses it.

Reply:
I found a doc here: https://github.com/netdata/netdata/blob/master/docs/guides/using-host-labels.md#host-labels-in-streaming
so maybe users are already using this.

Reply:
I agree with @mfundul , this looks like a compatibility break? The rest of the changes is right.

Reply:
No worries, nobody will open an issue to return masters and slaves in 2020 😉 🤣

Reply:
I've discussed it with @cakrit and agreed that the chances of people using it are minor and we are changing the terminology.